src/examples/theme-provider/ThemeProviderBranding.tsx

```diff
@@ -8,11 +8,11 @@
 import React from 'react';
 import { Row, Col } from '@zendeskgarden/react-grid';
 import { Button } from '@zendeskgarden/react-buttons';
-import { DefaultTheme, ThemeProvider } from '@zendeskgarden/react-theming';
+import { IGardenTheme, ThemeProvider } from '@zendeskgarden/react-theming';
```

Comment:
@austingreendev please double-check that I got this usage right throughout :pray:

Reply:
With the changes already in `styled.d.ts` I don't believe we need to retrieve `IGardenTheme` here. You should be able to use the following instead.

```
import { DefaultTheme } from 'styled-components';   
```

Reply:
Wasn't sure whether to optimize for `website` or `codesandbox`. But this makes sense.

Reply:
Within codesandbox the `DefaultTheme` value is `any` when it's not extended. So it should continue to work fine!